src/main/resources/rest-api-spec/api/bulk.json

```diff
@@ -0,0 +1,102 @@
+{
+  "bulk":{
```

Comment:
This file seems to be part of core. Is there any way to run test without copying non-plugin files?

Reply:
The tests are throwing exceptions if the dependent spec files are removed. I don't think we can run the tests without copying these specs. Please let me know if there is any feasible way to do it.

If we need to copy these specs then we should take care of these duplicate specs by skipping them while building the artifact by merging the specs from all plugins.

Reply:
yeah agree with @VijayanB. There seem to be definitely duplication interms of copying the keywords and other common stuff across plugins. It would be good to have these packaged part of some common utils library or may be part of the ```specifications``` package and get imported into the plugins. Looks like we need to invest more on this. But for now, can we put these common files into a separate folder and call something like ```utils``` and refer from there? This way it would be easy to migrate tomorrow.

Reply:
Actually there is a plugin called 'opensearch.rest-resources'. I tried to use it, but it didn't work as expected. 
https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/test/rest/RestResourcesPlugin.java

I will try to copy the common specs into a separate folder and use them to run tests.